config.yml.default

```diff
@@ -136,16 +136,16 @@ challenge:                   # Incoming challenges.
 #   - kingOfTheHill
 #   - racingKings
 #   - threeCheck
-  time_controls:             # Time controls to accept.
+  time_controls:                   # Time controls to accept.
     - bullet
     - blitz
     - rapid
-#   - classical
+    - classical
```

Comment:
I don't have a problem with this change.

Reply:
Okay